v03_pipeline/lib/reference_datasets/gnomad.py

```diff
@@ -0,0 +1,56 @@
+import hail as hl
+
+from v03_pipeline.lib.model import ReferenceGenome
+
+
+def select_fields_v2(ht: hl.Table) -> hl.Table:
+    selects = {}
+    global_idx = hl.eval(ht.globals.freq_index_dict['gnomad'])
+    selects['AF'] = hl.float32(ht.freq[global_idx].AF)
```

Comment:
unless you are conditionally modifying what select to run its more readable to directly call the select fields than assign to a dict:
```
ht = ht.select(
    AF=hl.float32(ht.freq[global_idx].AF),
    AN= ht.freq[global_idx].AN,
    ...
```

Reply:
yeah, I think we should move away from selecting from a dictionary of selects!